src/components/pages/profile/[name]/Profile.tsx

```diff
@@ -78,15 +77,12 @@ const ProfileContent = ({ nameDetails, isSelf, isLoading, name }: Props) => {
     basicIsCachedData,
     profileIsCachedData,
     wrapperData,
-    isWrapped,
+    canBeWrapped: _canBeWrapped,
   } = nameDetails
 
   const selfAbilities = useSelfAbilities(address, ownerData, name)
-  const nameWrapperExists = useWrapperExists()
   const canBeWrapped =
-    nameWrapperExists &&
-    !isWrapped &&
-    normalisedName.endsWith('.eth') &&
+    _canBeWrapped &&
```

Comment:
shouldn't the 'private' variable be the one that is only used in this component? not the one coming from nameDetails